XamlControlsGallery/ControlPagesSampleCode/ListView/ListViewSample4_xaml.txt

```diff
@@ -0,0 +1,26 @@
+<!-- XAML Code -->
+
+<!-- The ListView's ItemsSource is an ObservableCollection<Contact> that is defined and bound
+in the code-behind, called contactsFiltered. -->
+
+<ListView 
+    x:Name="FilteredListView"
+    ItemTemplate="{StaticResource ContactListViewTemplate}"
```

Comment:
Just a thought: Should we mention that the template can be found at the top, or should we expect that people will notice that?

Reply:
I'll add a note that it's the same template being used in the other samples above it 👍